config/install/field.field.node.localgov_page.localgov_page_content.yml

```diff
@@ -24,18 +32,26 @@ settings:
   handler: 'default:paragraph'
   handler_settings:
     target_bundles:
+      localgov_accordion: localgov_accordion
+      localgov_call_out_box: localgov_call_out_box
+      localgov_featured_teasers: localgov_featured_teasers
+      localgov_ia_block: localgov_ia_block
       localgov_image: localgov_image
       localgov_link: localgov_link
+      localgov_media_with_text: localgov_media_with_text
+      localgov_quote: localgov_quote
+      localgov_table: localgov_table
       localgov_text: localgov_text
-      localgov_page_section: localgov_page_section
```

Comment:
Looks like your removing the `localgov_page_section` paragraph provided by this module in preference for the `page_section` field provided by Subsites. This creates a dependency on Subsites as well as Subsites Paragraphs.